### ISSUE TITLE
Auto save / back up while editing

### ISSUE THREAD

**Issue by actonDev** [orgzly/orgzly-android#589]
I was writing in the app for quite some with my bluetooth keyboard and i fell asleep.
When I woke up, I unlocked the phone and for one brief moment i saw the text i was writing and then it disappeared. the whole thing! I never applied the changes, so I lost everything.

- any idea why it happened?
- regular auto saves would be a way to fix this - and a way to recover between auto-saved versions

**Comment by nevenz** [comment]
Hmm I can't reproduce it, views should keep the content and note shouldn't be reloaded. But it sounds like it was.

Can you reproduce it every time you lock the device for example?

Which device is this on?

**Comment by actonDev** [comment]
Sorry for the delay, it was a Xiaomi Redmi 4. Didn't manage to get the same thing to happen even though I tried to replicate it. And didn't have any other issue till then.
But would you be up for auto saves? another scenario would be if the phone dies from battery, or you accidentally close the app etc. Supposing that you're an emacs user, you know the value of auto-saves :)

If it's not in the schedule then this issue should just be closed.

**Comment by nevenz** [comment]
> But would you be up for auto saves?

Yes, I thought that there's already an issue for that, but apparently not.

**Comment by kidsil** [comment]
I've had the same issue, editing some text while opening Orgzly from a shortcut deleted all my edits.
